src/main/groovy/edu/oregonstate/mist/osuevents/db/LocalistDAO.groovy

```diff
@@ -0,0 +1,455 @@
+
+            edu.oregonstate.mist.osuevents.core.Department.fromLocalistDepartment(
+                    department.department)
+        }
+    }
+
+    /**
+     * Get all filters, including event types, event topics, audiences, and counties.
+     * @return
+     */
+    Filters getFilters() {
+        HttpResponse response = getResponse(filtersEndpoint)
+
+        String responseEntity = EntityUtils.toString(response.entity)
+
+        objectMapper.readValue(responseEntity, Filters)
+    }
+
+    /**
+     * Sends HTTP GET request. Accepts pagination parameters.
+     * @param endpoint
+     * @param pageNumber
+     * @param pageSize
+     * @return
+     */
+    private HttpResponse getResponse(String endpoint,
+                                     Integer pageNumber = null,
+                                     Integer pageSize = null) {
+        UriBuilder uriBuilder = UriBuilder.fromUri(baseURI)
+        uriBuilder.path(endpoint)
+
+        if (pageNumber != null) {
+            uriBuilder.queryParam("page", pageNumber)
+        }
+
+        if (pageSize != null) {
+            uriBuilder.queryParam("pp", pageSize)
+        }
+
+        URI requestURI = uriBuilder.build()
+
+        logger.info("Making a request to ${requestURI}")
+
+        httpClient.execute(new HttpGet(requestURI))
+    }
+
+}
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+class Filters {
+    @JsonProperty("event_counties")
+    List<Filter> counties
+
+    @JsonProperty("event_audience")
+    List<Filter> audiences
+
+    @JsonProperty("event_types")
+    List<Filter> eventTypes
+
+    @JsonProperty("event_event_topic")
+    List<Filter> eventTopics
+}
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+class Filter {
+    @JsonProperty("id")
+    String filterID
+
+    String name
+
+    @JsonProperty("parent_id")
+    String parentID
+}
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+class MultipleCommunitiesResponse {
+    List<SingleCommunityResponse> communities
+
+    @JsonProperty("page")
+    PaginationObject paginationObject
+}
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+class SingleCommunityResponse {
+    Community community
+}
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+class Community {
+    String id
+    String name
+
+    @JsonProperty("time_zone")
+    String timeZone
+
+    @JsonProperty("localist_url")
+    String calendarURL
+}
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+class MultiplePlacesResponse {
+    List<SinglePlaceResponse> places
+
+    @JsonProperty("page")
+    PaginationObject paginationObject
+}
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+class SinglePlaceResponse {
+    Place place
+}
+
+@JsonIgnoreProperties(ignoreUnknown=true)
+class Place {
+    String id
+    String name
+
+    @JsonProperty("campus_id")
+    String campusID
+
+    String latitude
+    String longitude
+    String street
+    String city
+    String state
+    String zip
+
+    @JsonProperty("localist_url")
+    String calendarURL
+    String url
```

Comment:
A blank line here? It seems like you use blank lines to separate the filed with `@JsonProperty` with other fields?